accounting/localizations/getting_started.rst

```diff
@@ -0,0 +1,491 @@
+==========================
+Making my Own Localization
+==========================
+
+.. warning::
+
+    This tutorial requires knownledges about how to build a module in Odoo.
+
+Building a localization module
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+A **Localization** module is an Odoo module containing all the necessary stuff to make Odoo suitable for a general purpose use in a particular country (e.g. charts of accounts, taxes, financial reports, etc).
+
+The first step is to create a new Odoo module named **l10n_xx** where **xx** is the lower case country code, taken from https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2.
+Obviously, this newly created module must depends of the ``account`` module. This dependency must be added to the ``__manifest__.py`` file.
+
+.. code-block:: py
+
+    ...
+    'depends' : ['account'],
+    ...
+
+There are two ways to install a **l10n_xx** module.
```

Comment:
I'm not sure it's worth giving all these technical details... I'd simply say that the localization module is installed based on the country, with fallback on the generic

```diff
-There are two ways to install a **l10n_xx** module.
-
-The first way is to install accounting while being certain that the right country is set on your company.
-Then, the localization module corresponding to the country code of the company will be installed automatically.
+Thanks to the``_auto_install_l10n()`` method defined on ``account.__init__.py``, the localization module corresponding to the country code of the company will be installed automatically upon the installation of the account module.
-This behavior is allowed by the ``_auto_install_l10n()`` method defined on ``account.__init__.py``.
In case of no country code set or no localization module found, the ``l10n_generic_coa`` (US) localization module will be installed by default.
-
-For more details about how this trick is possible inside the ``account`` module, see the python code below.
-
-*addons/account/__manifest__.py*
-
-.. code-block:: py
-
-   # -*- coding: utf-8 -*-
-   # Part of Odoo. See LICENSE file for full copyright and licensing details.
-   {
-       'name' : 'Invoicing',
-       'version' : '1.1',
-       'summary': 'Send Invoices and Track Payments',
-       'sequence': 30,
-       'description': """
-   Core mechanisms for the accounting modules. To display the menuitems, install the module account_invoicing.
-       """,
-       'category': 'Accounting',
-       'website': 'https://www.odoo.com/page/billing',
-       'images' : ['images/accounts.jpeg','images/bank_statement.jpeg','images/cash_register.jpeg','images/chart_of_accounts.jpeg','images/customer_invoice.jpeg','images/journal_entries.jpeg'],
-       'depends' : ['base_setup', 'product', 'analytic', 'web_planner', 'portal'],
-       'data': [
-           ...
-       ],
-       'demo': [
-           ...
-       ],
-       'qweb': [
-           ...
-       ],
-       'installable': True,
-       'application': False,
-       'auto_install': False,
-
-        # The localization module is installed thanks to the below line
-       'post_init_hook': '_auto_install_l10n',
-   }
-
-*addons/account/__init__.py*
-
-.. code-block:: py
-
-    # -*- coding: utf-8 -*-
-    # Part of Odoo. See LICENSE file for full copyright and licensing details.
-
-    def _auto_install_l10n(cr, registry):
-        # Search for a l10n_<country_code> localization module to install
-        ...
-        country_code = env.user.company_id.country_id.code
-        if country_code:
-            module_list = []
-            ...
-            if env['ir.module.module'].search([('name', '=', 'l10n_' + country_code.lower())]):
-                module_list.append('l10n_' + country_code.lower())
-            else:
-                module_list.append('l10n_generic_coa')
-            ...
-            module_ids = env['ir.module.module'].search([('name', 'in', module_list), ('state', '=', 'uninstalled')])
-            module_ids.sudo().button_install()
-
-An alternative of this trick is to install directly the localization module you want by searching it directly in the ``apps`` module:
-
-.. image:: media/getting_started01.png
-    :align: center
```